userspace/test_harness/src/lib.rs

```diff
@@ -17,10 +17,13 @@
 // Relies on internal details of rustc, so this may break during toolchain
 // updates.
 
+#![feature(asm)]
```

Comment:
Where is `feature(asm)` used in `test_harness`? Is it used by one of the new `libtock-rs` macros?

Reply:
Sorry about that, that got in there by mistake. Good catch. Removed.